lib/chat/ChatPage.dart

```diff
@@ -142,13 +143,17 @@ class _ChatPageState extends State<ChatPage> {
       viewModels.add(
         FarmDetailsListItemViewModel(
           title: value.title,
-          detail: value.value,
+          detail: _getDetail(value.value),
```

Comment:
maybe change value var name to "response" to avoid the value.value (did did the same thing on a recent PR)